src/UBAFeeCalculator/UBAFeeUtility.ts

```diff
@@ -303,6 +303,32 @@ export function calculateUtilization(
   return BigNumber.from(10).pow(decimals).sub(result);
 }
 
+export function calculationUtilizationBoundaries(
+  action: {
+    actionType: "deposit" | "refund";
+    amount: BigNumber;
+    chainId: number;
+  },
+  decimals: number,
+  hubBalance: BigNumber,
+  hubEquity: BigNumber,
+  ethSpokeBalance: BigNumber,
+  spokeTargets: { target: BigNumber; spokeChainId: number }[]
+): { utilizationPostTx: BigNumber; utilizationPreTx: BigNumber } {
+  let newEthSpokeBalance = ethSpokeBalance;
+  if (action.chainId === HUBPOOL_CHAIN_ID) {
+    if (action.actionType === "deposit") {
+      newEthSpokeBalance = newEthSpokeBalance.add(action.amount);
+    } else {
+      newEthSpokeBalance = newEthSpokeBalance.sub(action.amount);
+    }
+  }
+  return {
+    utilizationPreTx: calculateUtilization(decimals, hubBalance, hubEquity, ethSpokeBalance, spokeTargets),
+    utilizationPostTx: calculateUtilization(decimals, hubBalance, hubEquity, newEthSpokeBalance, spokeTargets),
+  };
+}
+
```

Comment:
This generates the initial bounds in order to calculate the system fee.

This is the `u_pre` and `u_post` defined [here](https://github.com/UMAprotocol/UMIPs/blob/680523cbbcb7a896abbe3d60504861ceeabeef29/UMIPs/umip-157.md#computing-the-lp-fee)